src/components/InfoIcon/InfoIcon.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { forwardRef, useRef, ReactNode } from "react";
+import { useMergeRefs } from "../../hooks";
+import VibeComponentProps from "../../types/VibeComponentProps";
+import VibeComponent from "../../types/VibeComponent";
+import { getTestId } from "../../tests/test-ids-utils";
+import { ComponentDefaultTestId } from "../../tests/constants";
+import Tooltip from "../Tooltip/Tooltip";
+import Icon from "../Icon/Icon";
+import { Info } from "../Icon/Icons";
+
+export interface InfoIconProps extends VibeComponentProps {
+  children: ReactNode;
+}
+
+const InfoIcon: VibeComponent<InfoIconProps, HTMLElement> = forwardRef(
+  ({ children, className, id, "data-testid": dataTestId }, ref) => {
+    const componentRef = useRef(null);
+    const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
+    return (
+      <span
```

Comment:
Tooltip is not a vibe component and doesn't get a ref so I had to wrap it in a span